web-extension/scripts/load-bookmarks.js

```diff
@@ -79,28 +78,15 @@ BmcMangaList.prototype.onSourceClick = function(comic, source) {
         resource: {
             origin: window.location.origin,
             reader: source.reader,
+            comicId: comic.id,
             comic: Object.assign({
                 common: {
                     name: source.name,
-                    chapter: comic.chapter,
-                    page: comic.page,
                 },
             }, source.info),
         },
     };
-    compat.sendMessage(ev, (err, response) => {
-        if (err) {
-            LOGS.warn('E0013', {'err': err});
-            return undefined;
-        }
-        let localEv = {
-            type: 'action',
-            action: 'urlopen',
-            url: response.resource.url,
-        };
-        // Let the content script at the page's root handle the URL opening
-        window.top.postMessage(localEv, '*');
```

Comment:
if you delete this message, you might as well delete the handling of said message in the relevant file.